pkg/apis/discovery/v1alpha1/clusterducktype_types.go

```diff
@@ -226,6 +234,9 @@ type ClusterDuckTypeStatus struct {
 
 	// DuckCount is the count of unique duck types found post-hunt.
 	DuckCount int `json:"duckCount"`
+
+	//ClusterRole Aggregation Rule
+	ClusterRoleAggregationRule rbacv1.AggregationRule `json:"clusterRoleAggregationRule,omitempty"`
```

Comment:
can you show what this would look like in a running resource? Also likely needs to be a pointer

Reply:
ah it is in the description: 

```
  clusterRoleAggregationRule:
    clusterRoleSelectors:
    - matchLabels:
        duck.knative.dev/channelable: "true"
```

I wonder if that helps anyone looking at this... I need to think

Reply:
I like it.